.github/workflows/build-pipeline.yml

```diff
@@ -112,7 +111,7 @@ jobs:
               --severity-threshold=high
               --fail-on=all
       - name: Run Snyk on Python
-        uses: snyk/actions/python-3.8@master
+        uses: snyk/actions/python-3.9@master
```

Comment:
should this be 3.10?

Reply:
3.10 wasn't there 2 weeks ago! haha yeah I'll update it